java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -1239,48 +1281,48 @@ public Builder appendUTF8String(byte[] value, int offset, int length) {
 
     public Builder appendArray(byte... values) {
       assert (values.length + currentIndex) <= rows;
-      assert type == DType.INT8 || type == DType.UINT8 || type == DType.BOOL8;
-      data.setBytes(currentIndex * type.sizeInBytes, values, 0, values.length);
+      assert type.typeId == DType.INT8 || type.typeId == DType.UINT8 || type.typeId == DType.BOOL8;
+      data.setBytes(currentIndex * type.typeId.sizeInBytes, values, 0, values.length);
       currentIndex += values.length;
       return this;
     }
 
     public Builder appendArray(short... values) {
-      assert type == DType.INT16 || type == DType.UINT16;
+      assert type.typeId == DType.INT16 || type.typeId == DType.UINT16;
```

Comment:
same comment here. can we have a helper?

Reply:
Done